test/sqlsrv/isPooled.php

```diff
@@ -0,0 +1,25 @@
+<?php
+include_once 'autonomous_setup.php';
+$conn1 = sqlsrv_connect( $serverName, $connectionInfo);
+$connId1 = ConnectionID($conn1);
+$conn1 = null;
+
```

Comment:
any reason you set $conn1 to null?
did you mean to use sqlsrv_close()?
same with stmt in ConnectionID() below... sqlsrv_free_stmt() should be used